sa3ed_app/www/found_person_create_detail/index.js

```diff
@@ -1,6 +1,23 @@
 var app = Vue.createApp({
     delimiters: ['[[', ']]'], // Change delimiters here
     data() {
+        frappe.call({
+            method:"sa3ed_app.api.WhitelistBypass.get_country_list",
+            callback: function(result) { 
+                message = result.message;
+                var select = document.getElementById("country_select");
```

Comment:
Use `let` or `const` instead of `var` depending on the context.
[Var, Let, and Const – What's the Difference?](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)

Reply:
This issue should be solved by Commit 2c03058